### PR TITLE
Improve(datahub_project): Improve resource use common method.

### PR DESCRIPTION
**Pull request by Lexsss**

### CODE REVIEW

File: alicloud/resource_alicloud_datahub_project_test.go [hunk 1]
```diff
@@ -111,70 +109,118 @@ func testSweepDatahubProject(region string) error {
...
+	testAccCheck := rac.resourceAttrMapUpdateSet()
+	rand := acctest.RandIntRange(100000, 999999)
+	name := fmt.Sprintf("tf_testaccdatahubproject%d", rand)
+	testAccConfig := resourceTestAccConfigFunc(resourceId, name, resourceDatahubProjectConfigDependence)
 
-func TestAccAlicloudDatahubProject_Update(t *testing.T) {
-	randdom := acctest.RandIntRange(datahubProjectSuffixMin, datahubProjectSuffixMax)
 	resource.Test(t, resource.TestCase{
 		PreCheck: func() {
-			testAccPreCheckWithRegions(t, true, connectivity.DatahubSupportedRegions)
```
Comment (YY-tomorrow) [inline alicloud/resource_alicloud_datahub_project_test.go]: Please keep this

File: alicloud/resource_alicloud_datahub_project_test.go [hunk 2]
```diff
@@ -111,70 +109,118 @@ func testSweepDatahubProject(region string) error {
...
+	rac := resourceAttrCheckInit(rc, ra)
+
+	testAccCheck := rac.resourceAttrMapUpdateSet()
+	rand := acctest.RandIntRange(100000, 999999)
+	name := fmt.Sprintf("tf_testaccdatahubproject%d", rand)
+	testAccConfig := resourceTestAccConfigFunc(resourceId, name, resourceDatahubProjectConfigDependence)
+
 	resource.Test(t, resource.TestCase{
 		PreCheck: func() {
-			testAccPreCheckWithRegions(t, true, connectivity.DatahubSupportedRegions)
```
Comment (YY-tomorrow) [inline alicloud/resource_alicloud_datahub_project_test.go]: Please keep this

File: alicloud/resource_alicloud_datahub_project_test.go [hunk 3]
```diff
@@ -111,70 +109,118 @@ func testSweepDatahubProject(region string) error {
...
+func resourceDatahubProjectConfigDependence(name string) string {
+	return ""
+}
+
+var datahubProjectBasicMap = map[string]string{
+	"name":    CHECKSET,
+	"comment": "project added by terraform",
+}
 
 func testAccCheckDatahubProjectExist(n string) resource.TestCheckFunc {
```
Comment (YY-tomorrow) [inline alicloud/resource_alicloud_datahub_project_test.go]: please remove this

File: alicloud/service_alicloud_datahub.go [hunk 4]
```diff
@@ -5,9 +5,55 @@ import (
...
+		if err != nil {
+			if NotFoundError(err) {
+				if status == Deleted {
+					return nil
+				}
+			} else {
+				return WrapError(err)
+			}
+		}
+
```
Comment (YY-tomorrow) [inline alicloud/service_alicloud_datahub.go]: please completion id or status

File: alicloud/resource_alicloud_datahub_project_test.go (hunk 3 above)
Comment (Lexsss) [reply alicloud/resource_alicloud_datahub_project_test.go]: still used

File: alicloud/service_alicloud_datahub.go (hunk 4 above)
Comment (Lexsss) [reply alicloud/service_alicloud_datahub.go]: no projectName or status returned

**Pull request merged**